content/tutorials/build-apps/build-a-browser-wallet-in-js.md

```diff
@@ -0,0 +1,210 @@
+
+Amount: 9
+
+Destination Tag: (Not usually necessary unless you're paying an account tied to an exchange)
+
+![Send XRP Transaction Screenshot](img/js-wallet-send-xrp-transaction-details.png)
+
+### 4. Create the Transactions Page
+
+Now that we have created the home page and the send XRP page, let's create the transactions page that will display the transaction history of the account. In order to see what's happening on the ledger, we're going to display the following fields: 
+
+  - Account: The account that sent the transaction.
+  - Destination: The account that received the transaction.
+  - Amount: The amount of XRP sent in the transaction.
+  - Transaction Type: The type of transaction.
+  - Result: The result of the transaction.
+  - Link: A link to the transaction on the XRP Ledger Explorer.
+
+![Transactions Page Screenshot](img/js-wallet-transaction.png)
+
+1. Create a folder named `transaction-history` in the src directory.
+2. Create a file named `transaction-history.js` and copy the code written below. 
+
+{{ include_code("_code-samples/build-a-wallet/js/src/transaction-history/transaction-history.js", language="js") }}
+
+This code uses [account_tx](account_tx.html) to fetch transactions we've sent to and from this account. In order to get all the results, we're using the `marker` parameter to paginate through the incomplete list of transactions until we reach the end.
+
+3. Create a file named `transaction-history.html` and copy the code from [transaction-history.html]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/js/src/transaction-history/transaction-history.html) into it.
+
+`transaction-history.html` defines a table which displays the fields mentioned above.
+
+You can use this code as a starting point for displaying your account's transaction history. If you want an additional challenge, try expanding it to support different transaction types (e.g. [TrustSet](trustset.html)). If you want inspiration for how to handle this, you can check out the [XRP Ledger Explorer](https://livenet.xrpl.org/) to see how the transaction details are displayed.
+
+## Next Steps
+
+Now that you have a functional wallet, you can take it in several new directions. The following are a few ideas:
+
+- You could support more of the XRP Ledger's [transaction types](transaction-types.html) including [tokens](issued-currencies.html) and [cross-currency payments](cross-currency-payments.html)
+- You could add support for displaying multiple tokens, beyond just XRP
+- You could support creating [offers](offers.html) in the [decentralized exchange](decentralized-exchange.html)
+- You could add new ways to request payments, such as with QR codes or URIs that open in your wallet.
+- Or you could support better account security including allowing users to set [regular key pairs](cryptographic-keys.html#regular-key-pair) or handle [multi-signing](multi-signing.html).
+
```

Comment:
It could be good to have a bullet point directing people to the docs for [Building for Production](https://vitejs.dev/guide/build.html#public-base-path) with Vite, too.

Reply:
Added